packages/api/src/modules/users/actions/updateDescription.ts

```diff
@@ -0,0 +1,17 @@
+import User from "../types/User";
+import knex from "../../../../db/knex";
+
+export default async <T extends keyof Omit<User, "password">>(
+    property: T,
+    value: User[T],
```

Comment:
Seems like you always reuse the findUser code, why do you think the function is there? Just make it accept an ID